apt/BUILD

```diff
@@ -22,7 +22,12 @@ load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
 bzl_library(
     name = "lib",
     srcs = [
-        "rules.bzl"
+        "rules.bzl",
+        "@rules_pkg//:pkg.bzl",
+        "@rules_pkg//pkg:pkg.bzl",
+    ],
+    deps = [
+        "@rules_pkg//doc_build:rules_pkg_lib",
```

Comment:
Pull in dependencies required for stardoc's `bzl_library` to run
In this case, `rules_pkg` actually already provides a `doc` package with a `bzl_library` we can depend on